lightningd/lightningd.h

```diff
@@ -80,6 +80,8 @@ struct lightningd {
 	char *config_dir;
 	char *rpc_filename;
 
+	/* Configuration file name */
+	char *config_filename;
```

Comment:
You need to initialize this to NULL in lightningd.c new_lightningd(), too.

Reply:
Thanks, fixed all